app/src/main/res/layout/fragment_attendee.xml

```diff
@@ -35,6 +35,36 @@
                 android:textSize="@dimen/text_size_large"
                 tools:text="May 24 - May 25" />
 
+            <LinearLayout
+                android:id="@+id/timeoutCounterLayout"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_marginTop="@dimen/layout_margin_large"
+                android:padding="@dimen/padding_large"
+                android:background="@drawable/border_background_ticket_timer"
+                android:visibility="gone"
+                tools:visibility="visible">
+                <TextView
+                    android:id="@+id/timeoutTextView"
+                    android:layout_weight="2"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:textSize="@dimen/text_size_expanded_title_large"
+                    android:textColor="@android:color/black"
+                    android:textStyle="bold"
+                    android:gravity="center_vertical"
+                    android:text="X:XX"/>
+                <TextView
+                    android:id="@+id/timeoutInfoTextView"
+                    android:layout_weight="8"
+                    android:layout_width="0dp"
+                    android:gravity="center_vertical"
+                    android:layout_height="match_parent"
+                    android:textSize="@dimen/text_size_large"
+                    android:text="After X minutes, the reservation we are hodling will be released"/>
```

Comment:
Hardcoded string!

Reply:
updated, I deleted the text in xml file anyway because it is not really necessary. When the timeout section is shown, new text is set.